plotting.py

```diff
@@ -0,0 +1,61 @@
+from earthquakes import get_data
+
+from datetime import date
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+def get_year(earthquake):
+    """Extract the year in which an earthquake happened."""
+    timestamp = earthquake['properties']['time']
+    # The time is given in a strange-looking but commonly-used format.
+    # To understand it, we can look at the documentation of the source data:
+    # https://earthquake.usgs.gov/data/comcat/index.php#time
+    # Fortunately, Python provides a way of interpreting this timestamp:
+    # (Question for discussion: Why do we divide by 1000?)
+    year = date.fromtimestamp(timestamp/1000).year
+    return year
+
+
+def get_magnitude(earthquake):
+    """Retrive the magnitude of an earthquake item."""
+    magnitude = earthquake['properties']['mag']
+
+    return magnitude
+
+
+# This is function you may want to create to break down the computations,
+# although it is not necessary. You may also change it to something different.
+def get_magnitudes_per_year(earthquakes):
+    """Retrieve the magnitudes of all the earthquakes in a given year.
+    
+    Returns a dictionary with years as keys, and lists of magnitudes as values.
+    """
+    ...
+
+
+def plot_average_magnitude_per_year(earthquakes):
+    ...
+
+
+def plot_number_per_year(earthquakes):
+    years = []
+    for i in quakes:
+        years.append(get_year(i))
```

Comment:
In general, try naming your variables more meaningfully, even if they only appear in a loop. If I see `for i in ...`, my first assumption is that it is a number counting from 0!